### ISSUE TITLE
Procedure

### ISSUE THREAD

**Issue by cbnestor** [AKS-Group/ax2012-infologs-fixes#1]
**Project: AKS_NC_ReportPLtoReportLayout**

`\Classes\PrintMgmtDocType`

![Image](https://github.com/AKS-Group/ax2012-infologs-fixes/assets/65027416/5d4ff36c-4ae2-4f22-ad6e-079f11fddd84)

For salesOrderInvoice

```
            //-->NC:11/10/2023 Original: 
            //    return ssrsReportStr(SalesInvoice, ReportPL);
            //ReportPL layout has been changed to Report for all PL companies
            return ssrsReportStr(SalesInvoice, Report);
            //<--NC:11/10/2023
```

![Image](https://github.com/AKS-Group/ax2012-infologs-fixes/assets/65027416/c6e4f4b0-0b18-4e68-b5fa-81109160f4cb)

For SalesPackingSlip

from 258 to 262 lines

```
                //-->AKS_NC:11/10/2023 Original
                //return ssrsReportStr(SalesPackingSlip, ReportPL);
                //ReportPL Layout changed to Report for Packing slip for all PL companies
                return ssrsReportStr(SalesPackingSlip, Report);
                //<--AKS_NC:11/10/2023
```

![Image](https://github.com/AKS-Group/ax2012-infologs-fixes/assets/65027416/a9d72526-e932-40be-9949-c5468aca775d)

**Form setup for Customer Invoice**

![Image](https://github.com/AKS-Group/ax2012-infologs-fixes/assets/65027416/d66a0756-b4b8-4d02-88ae-c869f2c58a18)

**Form Setup for Sales order Delivery Notes**

![Image](https://github.com/AKS-Group/ax2012-infologs-fixes/assets/65027416/7b2de041-15ba-43d1-966d-b19ec3c9f1c9)

**Reports layout Delivery note**

![Image](https://github.com/AKS-Group/ax2012-infologs-fixes/assets/65027416/0868d3dc-317e-41c2-a78a-d16e468b080c)

**Reports Layout for Cust sales Invoice**

![Image](https://github.com/AKS-Group/ax2012-infologs-fixes/assets/65027416/41416b40-5f8d-4a18-9543-3a512b2eaaa5)

**Comment by cbnestor** [comment]
`\Classes\PrintMgmtDocType`

For salesOrderInvoice

` //-->NC:11/10/2023 Original: 
  //    return ssrsReportStr(SalesInvoice, ReportPL);
  //ReportPL layout has been changed to Report for all PL companies
                return ssrsReportStr(SalesInvoice, Report);
  //<--NC:11/10/2023`

![Image](https://github.com/AKS-Group/ax2012-infologs-fixes/assets/65027416/26278619-7f43-4555-80bb-0cb57e5e1913)

**Comment by cbnestor** [comment]
On the test now the ReportPL has been replaced from the copy of Report






![Image](https://github.com/AKS-Group/ax2012-infologs-fixes/assets/65027416/fafff958-082f-4808-9704-bab3a61c0915)



![Image](https://github.com/AKS-Group/ax2012-infologs-fixes/assets/65027416/232863cf-399f-4fb7-8fa5-423529e2db1c)




![Image](https://github.com/AKS-Group/ax2012-infologs-fixes/assets/65027416/5712719d-c847-4a12-95db-30ace49acfcd)




![Image](https://github.com/AKS-Group/ax2012-infologs-fixes/assets/65027416/eed9a582-a0b0-4818-822b-1bf22359b4f6)




![Image](https://github.com/AKS-Group/ax2012-infologs-fixes/assets/65027416/e9d985e8-e4a0-4060-918d-6dc22c95ba1d)





![Image](https://github.com/AKS-Group/ax2012-infologs-fixes/assets/65027416/e45efca8-9c08-4de9-afa2-1b69cde69626)





![Image](https://github.com/AKS-Group/ax2012-infologs-fixes/assets/65027416/3e1c77c5-b99b-49e7-b575-2630254c199f)






![Image](https://github.com/AKS-Group/ax2012-infologs-fixes/assets/65027416/a782966a-26ab-4760-abb7-a7becc809291)
